src/main/resources/api/service_openapi.yaml

```diff
@@ -25,6 +25,19 @@ paths:
           description: Service is broken
           $ref: '#/components/responses/StatusResponse'
 
+  # TODO configure this to only be callable by Kubernetes in production.
```

Comment:
why we need this endpoint?

Reply:
Added a little more to the description of the endpoint, let me know if it's still not clear. Stairway in particular likes to shutdown gracefully so that less of its Flights are in a bad state.